contracts/monitoring/src/msg.rs

```diff
@@ -15,11 +15,14 @@ pub enum ExecuteMsg {
         chain_name: ChainName,
         new_prover_addr: Addr,
     },
+    RegisterActiveWorkerSet {
+        next_worker_set: WorkerSet,
```

Comment:
```suggestion
        worker_set: WorkerSet,
```
Also, do we really need the WorkerSet in monitoring, or can we just use the Addresses?

Reply:
I think we will need it in future...